hbogolib/handler.py

```diff
@@ -26,6 +26,8 @@
 except ImportError:
     from urllib.parse import unquote_plus as unquote
 
+import sqlite3
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [standard import "import sqlite3" should be placed before "import defusedxml.ElementTree as ET"](https://app.codacy.com/manual/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4949643)